pkgs/applications/virtualization/openstack/openstackclient/default.nix

```diff
@@ -0,0 +1,43 @@
+{ pkgs, stdenv, python2Packages }:
+
+let
+  python = { mkDerivation = pkgs.python2Packages.buildPythonPackage; };
+  generated = import ./requirements_generated.nix { inherit pkgs python; } ;
+in
+
+python2Packages.buildPythonApplication rec {
+  pname = "python-openstackclient";
+  version = "3.11.0";
+  name = "${pname}-${version}";
+
+  src = python2Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "79fc71a0234f212fc4cccb53b64763be7753e02c1666957a6210b16e8c68c2f6";
+  };
+
+  # We don't have build requires
+  doCheck = false;
+
+  propagatedBuildInputs = [
```

Comment:
yes, it would be better... but i get a strange error when I'm doing this:
'''
nix-build ./ -A openstackclient  --show-trace
error: while evaluating the attribute ‘propagatedBuildInputs’ of the derivation ‘python-openstackclient-3.11.0’ at /.../nixpkgs/pkgs/development/interpreters/python/mk-python-derivation.nix:56:3:
cannot coerce a function to a string, at /.../nixpkgs/pkgs/development/interpreters/python/mk-python-derivation.nix:56:3
'''
I don't know for now why these expressions are not equivalent.